cmd/cli/main_test.go

```diff
@@ -11,48 +11,44 @@ import (
 )
 
```

Comment:
:wrench: The file should be renamed to `args_test.go` as that's the logic it's testing